clients/da-vinci-client/src/main/java/com/linkedin/davinci/ingestion/DefaultIngestionBackend.java

```diff
@@ -84,7 +85,10 @@ public void startConsumption(VeniceStoreVersionConfig storeConfig, int partition
           storeVersion,
           partition);
     };
-    if (!storeAndVersion.getStore().isBlobTransferEnabled() || blobTransferManager == null) {
+
+    boolean blobTransferActiveInReceiver = shouldEnableBlobTransfer(storeAndVersion.getStore());
+
+    if (!blobTransferActiveInReceiver || blobTransferManager == null) {
```

Comment:
Here, it seems `blob.transfer.manager.enabled` is controlling both sender and receiver.
That means a node can't only run in receiver-only mode, which is fine.
It is good to call it out in the PR description.